src/redux/sagas/expenditures/lockExpenditure.ts

```diff
@@ -70,8 +69,6 @@ function* lockExpenditureAction({
       );
     }
 
-    yield put(transactionPending(lockExpenditure.id));
```

Comment:
Shouldn't we set the pending transaction also here?
`yield transactionSetPending(lockExpenditure.id);`